schemas/internal/storage.proto

```diff
@@ -16,10 +16,12 @@ enum LHStoreType {
 enum StoreableType {
     STORED_GETABLE = 0;
     SCHEDULED_TASK = 1;
-    WF_METRIC_UPDATE = 2;
+    reserved 2;
```

Comment:
I appreciate the thought here, this is the right thing to do for backwards compatibility in the future. It's optional (no pun intended) to do that now since `0.7.0` is breaking everything anyways.